synapse/rest/admin/users.py

```diff
@@ -66,12 +66,12 @@ class UsersRestServletV2(RestServlet):
     """
 
     def __init__(self, hs: "HomeServer"):
-        self.store = hs.get_datastore()
-        self.auth = hs.get_auth()
+        self._store = hs.get_datastore()
+        self._auth = hs.get_auth()
```

Comment:
This rename might be better served as a separate PR to keep the changes being proposed easy to track.